### PR TITLE
PART 2 `counter` smart contract in CosmWasm

### PR DESCRIPTION
**Pull request by DariuszDepta**
# Part 2
Implementation of the `counter` smart contract using pure ComsWasm libraries.

**ATTENTION!**
To be merged after merging **Part 1** #169

### CODE REVIEW

File: src/pages/cw-multi-test/getting-started/counter.mdx [hunk 1]
```diff
@@ -4,4 +4,4 @@ tags: ["multitest", "getting started", "counter"]
 
 # Counter
 
-(tbd)
+see: https://github.com/CosmWasm/docs/pull/169
```
Comment (jawoznia) [inline src/pages/cw-multi-test/getting-started/counter.mdx]: Just mark this PR as merge to the #169

**Review by jawoznia — COMMENTED** [review]
I think this documentation resembles too much the `cosmwasm-book`. I think it would be better to focus on the MultiTest functionality itself and leave CosmWasm examples to separate section.

File: src/pages/cw-multi-test/getting-started/counter.mdx (hunk 1 above)
Comment (jawoznia) [inline src/pages/cw-multi-test/getting-started/counter.mdx]: For future. please just rebase PR onto previous one

**Review by jawoznia — APPROVED** [review]

File: src/pages/cw-multi-test/getting-started/counter.mdx (hunk 1 above)
Comment (DariuszDepta) [reply src/pages/cw-multi-test/getting-started/counter.mdx]: Ok.

Comment (DariuszDepta) [reply src/pages/cw-multi-test/getting-started/counter.mdx]: Ok.

**Pull request merged**